backends/xnnpack/test/ops/test_rsqrt.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+from executorch.backends.xnnpack.test.tester import Tester
+
+
+class TestRsqrt(unittest.TestCase):
+    class Rsqrt(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+
+        def forward(self, x):
+            x = torch.abs(x)
+            z = torch.rsqrt(x)
+            return z
+
+    def _test_rsqrt(self, inputs):
+        (
+            Tester(self.Rsqrt(), inputs)
```

Comment:
Nit: Do you want to improve this test by adding dynamic_shape support like [this](https://github.com/pytorch/executorch/blob/019df698c4840adaef80cf39b7dd7ae42c9be6e5/backends/xnnpack/test/ops/test_softmax.py#L34).

Reply:
> improve this test by adding dynamic_shape support

it's a pointwise operator, why does it care about the shape? Everything is the same as sqrt including the test

Reply:
Yeah, just checking boxes, i.e. adding dynamic shape test for every operator.

Reply:
out of scope for this change! :)